evennia/commands/default/building.py

```diff
@@ -2315,6 +2315,8 @@ def func(self):
             else:
                 result = result[0]
                 string += "\n|g   %s - %s|n" % (result.get_display_name(caller), result.path)
+                if "locate" in self.cmdstring and not is_account and result.location:
```

Comment:
Previously 'locate' was just an alias. Now that it has an extra function, this needs to be indicated in the docstring of the Command..

Reply:
I see the point.  With some custom commands and aliases, I've documented the switch then mention which alias performs the switch by default, which makes the docstring much cleaner and adds functionality to the entire command by way of a switch instead.

```
    Usage:
      @find[/switches] <name or dbref or *account> [= dbrefmin[-dbrefmax]]
    Switches:
      room - only look for rooms (location=None)
      exit - only look for exits (destination!=None)
      char - only look for characters (BASE_CHARACTER_TYPECLASS)
      exact- only exact matches are returned.
      loc - also displays location of object if unique (same as @locate)
```

I'll make the doc change and code for this alteration if you agree.

Reply:
Actually, while adding a switch is fine, i think it makes sense to also include the alias in the Usage line, so
```
@find[/switches] <name or dbref or *account> [= dbrefmin[-dbrefmax]]
@locate [/switches] <name or dbref or *account> [= dbrefmin[-dbrefmax]]
```
And then add a note in the main description (and in the switches) that `@locate` is the same as giving `/loc` as a switch